src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,37 +1,99 @@
 import { useState } from 'react';
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
 
-export const NewMovie = () => {
-  // Increase the count after successful form submission
-  // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+export const NewMovie = ({ onAdd }: { onAdd: (newMovie: Movie) => void }) => {
+
+  const [title, setTitle] = useState('');
+  const [description, setDescription] = useState('');
+  const [imgUrl, setImgUrl] = useState('');
+  const [imdbUrl, setImdbUrl] = useState('');
+  const [imdbId, setImdbId] = useState('');
```

Comment:
Create only one state for movie
```suggestion
  const [movie, setMovie] = useState({
    title: '',
    .....
  });
```